src/game/objects/common.c

```diff
@@ -128,6 +128,7 @@ const GAME_OBJECT_ID g_TrapdoorObjects[] = {
     O_TRAPDOOR,
     O_TRAPDOOR2,
     O_BIGTRAPDOOR,
+    O_DRAW_BRIDGE,
```

Comment:
I think we can do one word "drawbridge". https://en.wikipedia.org/wiki/Drawbridge

It's usually spelled as one word (or with a hyphen according to Wikipedia).

Reply:
Gotcha, will adjust libtrx as well

Reply:
Done